src/cdk/overlay/position/position-strategy.ts

```diff
@@ -6,11 +6,17 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+import {OverlayRef} from '../overlay-ref';
+
+
 /** Strategy for setting the position on an overlay. */
 export interface PositionStrategy {
 
+  /** Attaches this position strategy to an overlay. */
+  attach(overlay: OverlayRef): void;
+
   /** Updates the position of the overlay element. */
-  apply(element: Element): void;
+  apply(): void;
```

Comment:
I'd rather leave them as is since normal use wouldn't chain them, just the tests